components/support/ktx/src/test/java/mozilla/components/support/ktx/android/content/res/Resources.kt

```diff
@@ -0,0 +1,19 @@
+package mozilla.components.support.ktx.android.content.res
+
+import android.util.DisplayMetrics
+import org.junit.Assert
+import org.junit.Test
+import org.robolectric.RuntimeEnvironment
+
+@Test
```

Comment:
Similar to `ViewTest` can we rename this file to `ResourcesTest` and create a test class for it?